### PR TITLE
Update CI to ignore rolling builds on 22.04 for PRs

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

**Comment by peterdavidfagan** [comment]
Hi @destogl, 

Thanks for committing these changes to the GitHub actions workflows.

I had the following question before completing the review (mainly for my own understanding):

1. Is there a reason why the workflows [1](https://github.com/ros-controls/ros2_control_demos/blob/ci-fixes/.github/workflows/rolling-semi-binary-build-last-focal.yml) and [2](https://github.com/ros-controls/ros2_control_demos/blob/ci-fixes/.github/workflows/rolling-binary-build-last-focal.yml) don't also have the pull request trigger commented out? (I presume this is intentional but wished to check)
 
2. Is there a reason for commenting out the trigger over removing this code?

It is awesome to have the opportunity to participate in code reviews for this project, thanks again for opening this up to the community. 

The above changes otherwise look good to me.

File: .github/workflows/rolling-binary-build.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ name: Rolling Binary Build
 # description: 'Build & test all dependencies from released (binary) packages.'
 
 on:
-  pull_request:
-    branches:
-      - master
+#  pull_request:
+#    branches:
+#      - master
```
Comment (destogl) [inline .github/workflows/rolling-binary-build.yml:8]:
```suggestion
  pull_request:
    branches:
      - master
```

**Comment by destogl** [comment]
>     1. Is there a reason why the workflows [1](https://github.com/ros-controls/ros2_control_demos/blob/ci-fixes/.github/workflows/rolling-semi-binary-build-last-focal.yml) and [2](https://github.com/ros-controls/ros2_control_demos/blob/ci-fixes/.github/workflows/rolling-binary-build-last-focal.yml) don't also have the pull request trigger code commented out within the workflow? (I presume this is intentional but wished to check)

Yes, the reason we are commenting the others out is that they always fail, since rolling on 22.04 is broken. As soon as it works, we will revert this. (It seems that this would happen anytime soon - [see discourse](https://discourse.ros.org/t/preparing-for-rolling-sync-2022-03-24/24521)).
> 
>     2. Is there a reason for commenting out the trigger rather than removing this code?

Just for simpler reverting.


> It is awesome to have the opportunity to participate in code reviews for this project, thanks again for opening this up to the community.

Thank you for contributing your time and knowledge into the project!!!!

File: .github/workflows/rolling-semi-binary-build.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ name: Rolling Semi-Binary Build
 # description: 'Build & test that compiles the main dependencies from source.'
 
 on:
-  pull_request:
-    branches:
-      - master
+#  pull_request:
+#    branches:
+#      - master
```
Comment (destogl) [inline .github/workflows/rolling-semi-binary-build.yml:7]:
```suggestion
  pull_request:
    branches:
      - master
```

**Review by Briancbn — APPROVED** [review]

**Review by bmagyar — APPROVED** [review]

**Pull request merged**